openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -459,9 +462,31 @@ def get_extended_profile(user_profile):
             "field_name": field_name,
             "field_value": extended_profile_fields_data.get(field_name, "")
         })
+    extended_profile.extend(get_custom_form_fields(user_profile.user, CustomFormFields))
     return extended_profile
 
 
+def get_custom_form_fields(user, custom_model):
```

Comment:
we could move this function to the plugin, see the get_fields method here https://bitbucket.org/edunext/llpa_openedx_extensions/src/5cea1513e41d3e67379a46065659c329888853be/llpa_openedx_extensions/custom_registration_form/models.py#lines-69